Update sensorpush to include note about activation being required (#23716)
home-assistant/home-assistant.github.io: source/_integrations/sensorpush.markdown

Integrates [SensorPush](https://www.sensorpush.com/) devices into Home Assistant.

> ## App Configuration Required To See Entities

Reviewer: ```suggestion
## Activation is required
```

> Sensor entities (temperature, humidity, barometric pressure) will not be available to Home Assistant until you have configured them from within the SensorPush app on iOS or Android.  
> 
> Using the SensorPush app to configure sensors does not require creation of a SensorPush account or the provision of any information. Communication between the sensor and Home Assistant will be purely local via BLE.

Reviewer: ```suggestion

Sensor entities (temperature, humidity, barometric pressure) will not be available to Home Assistant until you have activated the device with the SensorPush app on iOS or Android.
```

Let's not comment on if an account is required or not, because they could change that
